src/reminders/remind_router.erl

```diff
@@ -24,6 +25,12 @@ send_test(User, Channel, HolidayDate) ->
   {ok, Pid} = supervisor:start_child(remind_router_sup, []),
   gen_server:cast(Pid, {send_reminder, User, Channel, HolidayDate, Message, true}).
 
+send_message(User, Channel, Message) ->
+  UserName = maps:get(name, User),
+  FinalMessage = <<UserName/binary, ": ", Message/binary>>,
+  {ok, Pid} = supervisor:start_child(remind_router_sup, []),
+  gen_server:cast(Pid, {send_reminder, User, Channel, erlang:date(), FinalMessage, true}).
```

Comment:
another issue is that right now, the log of sent reminders in the database has a boolean column `test`. Now, apart from regular reminders and test reminders, we introduced these notifications. We should probably remove the test column and have a reminder type or something, to properly inform in the log.

Reply:
Yes, i believe that creating a "message type" or something like that would be the most flexible way of doing it.